### DISCUSSION TITLE
[BUG] Invalid list indentation

### DISCUSSION THREAD

**Discussion by pktiuk** [mkdocs/mkdocs#3063] [Q&A] [answered]
Instead of getting properly formatted multi-level list, I get only one-level

![image](https://user-images.githubusercontent.com/45544416/206550089-14619927-8e19-4f58-bfc2-97d6c3d88d7f.png)

To get proper indentation I have to use 4 spaces, but this is just a workaround. (in some cases regular lists with 2 spaces work fine)

## Config

```
$ mkdocs --version
mkdocs, version 1.4.2 
```

## Minimal example for reproduction

Files:

```
.
├── docs
│   └── haskell.md
└── mkdocs.yml

```

mkdocs.yml:

```yml
site_name: Notatki
```

docs/haskell.md:

```md
# Haskell λ

Jest to statycznie typowany, głównie deklaratywny, czysto funkcyjny język programowania.

Jego najpopularniejszym kompilatorem jest GHC (Glasgow Haskell Compiler).

## Zmienne

Większość przydatnych rzeczy jest w [dokumentacji](https://hackage.haskell.org/package/base-4.9.1.0/docs/Prelude)

- `Char` - znak
  - `isAlpha`, `isAlphaNum`, `isDigit`, `isLower`, `isUpper` :: **Char -> Bool**
  - `toLower`, `toUpper` :: **Char -> Char**
  - `putChar` :: **Char -> IO ()**
- `String` - `type String = [Char]`
- `Num`, `Int`, `Float` - liczba
  - `div`, `mod`

Do zdefiniowania zmiennej możemy użyć słowa kluczowego `let`
```

Run `mkdocs serve` and open `http://127.0.0.1:8000/haskell/#haskell`

**Comment by c-mart** [comment, marked as answer]
Consider https://pypi.org/project/mdx-truly-sane-lists/?

**Reply by pktiuk** [reply]
How am I supposed to use it with mkdocs?

**Reply by c-mart** [reply]
`pip install` it and configure your `mkdocs.yml` thusly:

```
markdown_extensions:
- mdx_truly_sane_lists
```

**Reply by pktiuk** [reply]
thanks
